lib/controllers/ServiceFabrikApiController.js

```diff
@@ -474,6 +475,63 @@ class ServiceFabrikApiController extends BaseController {
         .send({}));
   }
 
+  scheduleUpdate(req, res) {
+    req.manager.isAutoUpdatePossible();
+    if (_.isEmpty(req.body.repeatInterval)) {
+      throw new BadRequest('repeatInterval is mandatory');
```

Comment:
keep some reference from the request (e.g. instance id or / deployment name etc.), otherwise in huge log difficult to debug for multiple occurrence of this type.

Reply:
Not required. Exception that gets thrown from here gets logged by EventLogInterceptor which logs all the details related to current request. So we will not miss the context.